javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,133 @@
+// WARM UP TASK
+const names = [
+  'Peter',
+  'Ahmad',
+  'Yana',
+  'kristina',
+  'Rasmus',
+  'Samuel',
+  'katrine',
+  'Tala',
+];
+
+const nameToRemove = 'Ahmad';
+
+const indexToRemove = names.includes(nameToRemove);
+if (indexToRemove !== 1) {
+  names.splice(indexToRemove, 1);
+}
+
+console.log(names);
+
+//TASK1 : When will we be there??
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function calculateTravelTime(info) {
+  const totalHours = info.destinationDistance / info.speed;
+  const hours = Math.floor(totalHours);
+  const minutes = Math.round((totalHours - hours) * 60);
+  return `${hours} hours and ${minutes} minutes`;
+}
+
+const travelTime = calculateTravelTime(travelInformation);
+console.log(travelTime);
+
+//TASK2 : Series duration of my life
+const seriesDurations = [
+  {
+    title: 'Game of thrones',
+    days: 2,
+    hours: 20,
+    minutes: 8,
+  },
+  {
+    title: 'How to get away with murder',
+    days: 2,
+    hours: 16,
+    minutes: 30,
+  },
+  {
+    title: 'See',
+    days: 0,
+    hours: 21,
+    minutes: 12,
+  },
+  {
+    title: 'Peacky Blinders',
+    days: 1,
+    hours: 9,
+    minutes: 36,
+  },
+  {
+    title: 'Gossip girl',
+    days: 3,
+    hours: 12,
+    minutes: 42,
+  },
+];
+
+function logOutSeriesText() {
+  let averageLifespan = 80;
+  const averageLiveMinutes = averageLifespan * 365.25 * 24 * 60;
+  let totalPercentage = 0;
+
+  seriesDurations.forEach(series => {
```

Comment:
Exciting use of a higher-order function! Can you explain what `forEach` and `=>` does? 😄 

If you are comfortable with higher-order functions, you could switch to `seriesDurations.reduce(...)` here instead. 🙂

Reply:
yes, sure, so `forEach` is a method that goes through each item in the array and runs the function we give it for each item; 
and `=> ` is an arrow function used to write it quickly and shorter